projects/ccf-rui/src/app/shared/components/visibility-menu/visibility-menu.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="item-entry" [class.selected]="selection ? selection.id === item.id : false" *ngFor="let item of items; trackBy: getId" (mouseover)="mouseOver(item)" (mouseout)="mouseOut(item)">
```

Comment:
Reuse the getId function for class.selected. `getId(0, selection) === getId(0, item)`